tests/integration/test_data/pip_e2e_test/container/Containerfile

```diff
@@ -0,0 +1,20 @@
+# FROM quay.io/parent-ubi8-with-python3-pip  # TODO: Build and push it to Quay
+FROM docker.io/python:3.8
```

Comment:
Btw, if you _do_ need ubi8 with python3 and pip, a good choice could be registry.access.redhat.com/ubi8/python-39

Reply:
ou it's not necessary after all. It works fine with just simple image with python3.8, but thank you!